.github/workflows/test.yml

```diff
@@ -7,34 +7,31 @@ on:
   pull_request:
 
 env:
-  CI: true
-  NODE: 12.x
+  FORCE_COLOR: 2
+  NODE: 12
```

Comment:
Should this be 16? Or you guys want to only check the minimum supported version?

Reply:
Should be a matrix. We should be testing on all supported versions (12, 14, 16, 17) IMO.